File: MPP_Problema1/Model/Flight.cs
using Npgsql;
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.Data;

namespace MPP_Problema1.Model
{
    [Table("public.flight")]
    public class Flight
    {
        [Column("id")]
        public Guid Id { get; }

        [Column("departure")]
        public string Departure { get; }

        [Column("destination")]
        public string Destination { get; }

        [Column("departure_date_time")]
        public DateTime DepartureDateTime { get; }

        [Column("arrival_date_time")]
        public DateTime ArrivalDateTime { get; }

        [Column ("capacity")]
        public int Capacity { get; }
        public Flight(string departure, string destination, DateTime flightDate, DateTime flightTime, int capacity)
        {
            this.Id = Guid.NewGuid();
            this.Departure = departure;
            this.Destination = destination;
            this.DepartureDateTime = flightDate;
            this.ArrivalDateTime = flightTime;
            this.Capacity = capacity;
        }

        public Flight(Guid id, string departure, string destination, DateTime departureDateTime, DateTime arrivalDateTime, int capacity)
        {
            this.Id = id;
            this.Departure = departure;
            this.Destination = destination;
            this.DepartureDateTime = departureDateTime;
            this.ArrivalDateTime = arrivalDateTime;
            this.Capacity = capacity;
        }

        public Flight(object id, object departure, object destination, object departureDateTime, object arrivalDateTime, object capacity)
        {
            this.Id = (Guid)id;
            this.Departure = (string)departure;
            this.Destination = (string)destination;
            this.DepartureDateTime = (DateTime)departureDateTime;
            this.ArrivalDateTime = (DateTime)arrivalDateTime;
            this.Capacity = (int)capacity;

           
        }
    }
}

File: src/main/java/org/example/model/Flight.java
package org.example.model;

public class Flight{


    private String id;
    private String departure;
    private String destination;

    private String arrivalDateTime;

    private String departureDateTime;

    private int capacity;

//    public Flight(String id, String departure, String destination, String flight_date, String flight_time, int capacity) {
//        this.id=id;
//        this.departure = departure;
//        this.destination = destination;
//        this.flight_date = flight_date;
//        this.flight_time = flight_time;
//        this.capacity = capacity;
//    }

    public String getDeparture() {
        return departure;
    }

    public void setDeparture(String departure) {
        this.departure = departure;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }



    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getArrivalDateTime() {
        return arrivalDateTime;
    }

    public void setArrivalDateTime(String arrivalDateTime) {
        this.arrivalDateTime = arrivalDateTime;
    }

    public String getDepartureDateTime() {
        return departureDateTime;
    }

    public void setDepartureDateTime(String departureDateTime) {
        this.departureDateTime = departureDateTime;
    }

    @Override
    public String toString() {
        return "Flight{" +
                ", departure='" + departure + '\'' +
                ", destination='" + destination + '\'' +
                ", flight_date=" + arrivalDateTime +
                ", flight_time=" + departureDateTime +
                ", capacity=" + capacity +
                '}';
    }
}